js/ui/windowManager.js

```diff
@@ -19,6 +19,7 @@ const Tweener = imports.ui.tweener;
 const WindowMenu = imports.ui.windowMenu;
 
 const SHELL_KEYBINDINGS_SCHEMA = 'org.gnome.shell.keybindings';
+const NO_DEFAULT_MAXIMIZE_KEY = 'no-default-maximize';
```

Comment:
Please move this line to https://github.com/endlessm/gnome-shell/pull/5/commits/24adc95e3da6bf914c9ab21fe0542b91794e1c83